### PR TITLE
Apply isort linter rule

### PR DESCRIPTION
**Pull request by iamdefinitelyahuman**
### What I did
* Adjust `isort` linter rules to be compatible with `black`
* Apply `black` to `vyper/ast` and `vyper/compiler`
* Pin `flake8` dependency to `3.7.9` - `3.8.0` causes significant breaking changes

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
     'hypothesis[lark]>=4.53.2,<5',
 ]
 lint_deps = [
-    'flake8>=3.7,<4',
+    'flake8==3.7.9',
```
Comment (fubuloubu) [inline setup.py:22]:
```suggestion
    'flake8>=3.7.9,<3.8.0',
```

File: tests/parser/features/test_internal_call.py [hunk 2]
```diff
@@ -1,16 +1,9 @@
...
 
-from vyper.compiler import (
-    compile_code,
-)
-from vyper.exceptions import (
-    StructureException,
-    TypeMismatch,
-)
+from vyper.compiler import compile_code
+from vyper.exceptions import StructureException, TypeMismatch
```
Comment (fubuloubu) [inline tests/parser/features/test_internal_call.py:6]: This is what `isort` wants? I would think 2+ imports would be the other way

File: vyper/__init__.py [hunk 3]
```diff
@@ -2,10 +2,7 @@
 
 import pkg_resources as _pkg_resources
 
-from vyper.compiler import (  # noqa
-    compile_code,
-    compile_codes,
-)
+from vyper.compiler import compile_code, compile_codes  # noqa
```
Comment (fubuloubu) [inline vyper/__init__.py:5]: is `noqa` still needed?

File: tests/parser/features/test_internal_call.py (hunk 2 above)
Comment (iamdefinitelyahuman) [reply tests/parser/features/test_internal_call.py:6]: This is what `black` wants, in order to reduce the size of diffs.

File: vyper/__init__.py (hunk 3 above)
Comment (iamdefinitelyahuman) [reply vyper/__init__.py:5]: Yes, but it shouldn't be a blanket `# noqa` here it should be `# noqa: F401`.  Good catch.

Comment (iamdefinitelyahuman) [reply vyper/__init__.py]: Fixed

**Pull request merged**